.github/workflows/go-integration-test.yml

```diff
@@ -37,6 +37,11 @@ on:
         description: "build artifact to download before running tests"
         required: false
         type: string
+      runsOn:
+        description: "github actions runner to use for this action"
+        default: "ubuntu-latest"
```

Comment:
don't want to keep usng test-large-runner here?

Reply:
nah I had just changed it yesterday to test stuff, this is reverting back + allowing specific repos to still use large runners